docker-compose.yml

```diff
@@ -1,23 +1,21 @@
 version: '3'
 services:
     mysql:
-        image: mysql
-
-        environment:
-            - MYSQL_ROOT_PASSWORD=
+        build: docker-services/mysql/.
+        ## For Environment Variables, please see the Dockerfile in the mysql FOLDER, they weren't sending at the right spot for the scripts from here
 
     lidarrupdate:
         build: .
 
         ports:
-            - "5001:5000"
+            - "5000:5000"
 
         links:
             - mysql
 
         environment:
             - DataDirectory=/data
-            - Database="server=127.0.0.1;user id=root;password=${MYSQL_ROOT_PASSWORD};database=lidarrupdate;CharSet=utf8mb4"
+            - Database=server=mysql;user id=root;password=root;database=lidarrupdate;CharSet=utf8mb4
```

Comment:
Why take out root password by environment variable?

Reply:
And can we verify that the `Database` variable in the container is what we expect? I think it may be fine, but I'm worried that the equals and semicolons could cause some issues with the quotes taken out.